lib/cache/cache.go

```diff
@@ -376,9 +376,11 @@ func ForOkta(cfg Config) Config {
 		// the cache.
 		{Kind: types.KindAccessRequest},
 		{Kind: types.KindApp},
```

Comment:
Does Okta service need to watch for access requests and apps now? I thought we discussed that it doesn't really need access requests because it watches assignments directly? And dynamic apps it doesn't really need either?

Reply:
Ahh yes, good point. I meant to remove these!